src/test/test_nvmeof_mon_encoding.cc

```diff
@@ -163,6 +163,7 @@ void test_MNVMeofGwBeacon() {
 void test_NVMeofGwTimers()
 {
     NVMeofGwMap pending_map;
+    uint64_t features = 4540701547738038271;
```

Comment:
Same comment as above -- this should probably be CEPH_FEATURES_ALL.

Reply:
Also, since you'll need to edit this part anyway, just pass CEPH_FEATURES_ALL to encode -- there's no reason to use a temp variable.

Reply:
These two comments have not been addressed.

Reply:
fixed